autocomplete/definitions/namedTypes/mwseMCMSlider/convertToLabelValue.lua

```diff
@@ -0,0 +1,21 @@
+return {
+	type = "method",
+	description = [[This function specifies how values stored in the `variable` field should correspond to values displayed in the slider label.
+The default behavior is to consistently format decimal places (i.e., if `decimalPlaces == 2`, make sure two decimal places are shown.)
+This can be overwritten in the `createNewSlider` method, allowing for custom formatting of variable values.]],
+	arguments = {
+		{ name = "variableValue", type = "number" }
+	},
+    examples = {
```

Comment:
This function has a return value.

Reply:
looks like i got a little sloppy with the copy pasting last night. all three of these things got fixed. 

i went with your suggestion of not storing the results in the `convertToLabelValue` examples, i think that was a nice aesthetic choice.